contents/docs/self-host/deploy/configuration.md

```diff
@@ -122,53 +122,81 @@ plugins:
 
 To use an external `ClickHouse` cluster, set `clickhouseOperator.enabled` to `false` and set `clickhouse.host`, `clickhouse.database`, `clickhouse.user` and `clickhouse.password`.
 
-_See [ALL_VALUES.md](https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/ALL_VALUES.md) for full configuration options._
 
-### [PostgreSQL](https://www.postgresql.org/)
+### [PostgreSQL](../runbook/postgresql/)
 
 > While ClickHouse powers the bulk of the analytics if you deploy PostHog using this chart, Postgres is still needed as a data store for PostHog to work.
 
 By default, PostgreSQL is installed as part of the chart. To use an external PostgreSQL server set `postgresql.enabled` to `false` and then set `postgresql.postgresHost` and `postgresql.postgresqlPassword`. The other options (`postgresql.postgresqlDatabase`, `postgresql.postgresqlUsername` and `postgresql.postgresqlPort`) may also want changing from their default values.
 
 To avoid issues when upgrading this chart, provide `postgresql.postgresqlPassword` for subsequent upgrades. This is due to an issue in the PostgreSQL chart where password will be overwritten with randomly generated passwords otherwise. See [PostgreSQL#upgrade](https://github.com/helm/charts/tree/master/stable/postgresql#upgrade) for more detail.
 
-_See [ALL_VALUES.md](https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/ALL_VALUES.md) and [PostgreSQL chart](https://github.com/bitnami/charts/tree/master/bitnami/postgresql) for full configuration options._
 
 ### [Redis](https://redis.io/)
 
-By default, Redis is installed as part of the chart. To use an external Redis server/cluster set `redis.enabled` to `false` and then set `redis.host`. If your Redis cluster uses a password define it with `redis.password`, otherwise just omit it.
+Redis is installed by default as part of the chart. You can customize all its settings by overriding `values.yaml` variables in the `redis` namespace.
 
-To avoid issues when upgrading this chart, provide `redis.password` for subsequent upgrades. Otherwise, the Redis pods will get recreated on every update, potentially incurring some downtime.
+#### Use an external service
+To use an external Redis service, please set `redis.enabled` to `false` and then configure the `externalRedis` values.
 
-_See [ALL_VALUES.md](https://github.com/PostHog/charts-clickhouse/blob/main/charts/posthog/ALL_VALUES.md) and [redis chart](https://github.com/bitnami/charts/tree/master/bitnami/redis) for full configuration options._
+#### Credentials
+By default, Redis doesn't use any password for authentication. If you want to configure it to use a password (recommended) see the options below.
 
-### [Kafka](https://kafka.apache.org/)
+##### Internal Redis
```

Comment:
I think these sections make the document hard to follow.

As above, I suggest using `<details>` to make these sections collapsible - though we'd also lose them from the side nav.

Reply:
What about putting the content in <details> while keeping the paragraph title?

Reply:
If that works :+1: from me.

I don't think the loss of headers is a big deal though even if it happens.